core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1142,7 +1142,6 @@ protected static function replaceOldBePaths($strContext)
 			'contao/password.php'  => $generate('contao_backend_password'),
 			'contao/popup.php'     => $generate('contao_backend_popup'),
 			'contao/preview.php'   => $generate('contao_backend_preview'),
-			'contao/switch.php'    => $generate('contao_backend_switch')
```

Comment:
In this file there's still the `win.parent.postMessage` JS code which was used to communicate with the iframe previously. This can be removed as well now.

Reply:
@leofeyer can you review 7666ac6aa0b10026d14f5fca830432ed0306e7c8?

Reply:
This is correct 👍